configs/javascript.js

```diff
@@ -202,6 +202,22 @@ module.exports = {
 		],
 
 		'wrap-iife': [ 'error', 'any' ],
+
+		'promise/always-return': 'off',
+		'promise/avoid-new': 'off',
+		'promise/catch-or-return': 'off',
```

Comment:
`catch-and-return` seems sensible, any reason not to enable it?

Reply:
A lot of noise in case of `Promise<void>` :-(